workshops/personal_website/README.md

```diff
@@ -200,7 +219,7 @@ Go ahead and add this into your `index.html` now. I put my picture before my hea
 
 Hurray! You've created a basic HTML page! Let's save our progress with git!
 
-In Cloud9, press `Alt + t` on the keyboard. This will open the terminal, which allows us to run special types of commands in Cloud9. In this case, we're going to save the work we've done so far, using git. Type in the following commands:
+In Cloud9, press `Alt + t` (`option + t` on mac) on the keyboard. This will open the terminal, which allows us to run special types of commands in Cloud9. In this case, we're going to save the work we've done so far, using git. Type in the following commands:
```

Comment:
Ah, good point. I guess macOS = operating system, Mac = physical computer.

Can you capitalize Mac?

On Thu, Aug 11, 2016 at 7:14 PM, Max Wofford <notifications@github.com>
wrote:

> In workshops/personal_website/README.md
> <https://github.com/hackclub/hackclub/pull/885#discussion_r74533639>:
>
> > @@ -200,7 +219,7 @@ Go ahead and add this into your `index.html` now. I put my picture before my hea
> >
> >  Hurray! You've created a basic HTML page! Let's save our progress with git!
> >
> > -In Cloud9, press `Alt + t` on the keyboard. This will open the terminal, which allows us to run special types of commands in Cloud9. In this case, we're going to save the work we've done so far, using git. Type in the following commands:
> > +In Cloud9, press `Alt + t` (`option + t` on mac) on the keyboard. This will open the terminal, which allows us to run special types of commands in Cloud9. In this case, we're going to save the work we've done so far, using git. Type in the following commands:
>
> They use "Mac" on their website when referring to keyboards:
> https://support.apple.com/en-us/HT201236
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hackclub/hackclub/pull/885/files/42e6b499c0099d3715550084f7b84b0f2c666356#r74533639>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA8j-O3Etgz7pdY3WUOCF7Z_Jn0PI55pks5qe9cTgaJpZM4JgUDv>
> .
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>